Sources/App/routes.swift

```diff
@@ -4,22 +4,22 @@ import Vapor
 public func routes(
     _ router: Router,
     logger: LoggerProtocol,
-    mergeService: MergeService,
+    dispatchService: DispatchService,
     gitHubEventsService: GitHubEventsService
 ) throws {
 
     router.get("/") { request -> Response in
         let response = Response(using: request)
         if request.header(named: HTTPHeaderName.accept.description) == "application/json" {
-            try response.content.encode(mergeService.state.value, as: .json)
+            try response.content.encode(dispatchService.queues, as: .json)
```

Comment:
@ilyapuchka Please note that this will change the API used by the ClientApp you recently introduced, now returning a dictionary of `[String: MergeState.State]` indexed by the branch names – since we'll now have multiple Merge Queues in parallel – instead of what was previously just a single `MergeService.State`.

Reply:
Is `queues` property used for anything else? If not then it would be easier for paring to make `branch` a property of `MergeService.state` and encode in into json

Reply:
Nope `queues` is only used there – in fact I only introduced it in the merge commit when I tried to merge `master` into this branch and had to fix conflicts with your change about JSON queue, so it has introduced just for that.

Reply:
Fixed in 820623f